charts/clusterpedia/values.yaml

```diff
@@ -58,6 +58,16 @@ storageConfig:
     slowThreshold: 100ms
     ## @param storageConfig.log.ignoreRecordNotFoundError indicates whether ignore error type `NotFound`
     ignoreRecordNotFoundError: false
+    ## @param storageConfig.log.logger indicates the log rotate config
+    logger:
+      ## @param storageConfig.log.logger.filename indicates the file to write logs to
+      filename: /var/log/clusterpedia/internalstorage.log
+      ## @param storageConfig.log.logger.maxSize indicates the maximum size in megabytes of the log file before it gets
+      ## rotated
+      maxSize: 100
+      ## @param storageConfig.log.logger.maxBackups indicates the maximum number of old log files to retain. Default 0
+      ## is to retain all old log files
+      maxBackups: 0
```

Comment:
The default value of 1 in the code, what do you think is better

Reply:
> The default value of 1 in the code, what do you think is better

Thanks for reminding, let's keep the same with the default lumberjack logger behavior, which is `0`.